_methods/participatory-design.md

```diff
@@ -0,0 +1,38 @@
+---
+layout: card
+title: Participatory design
+permalink: /fundamentals/participatory-design/
+redirect_from:
+  - /participatory-design/
+description: Design processes that invite participation are more sustainable in the long run.
+category: Fundamentals
+what: An approach to design emphasizing shared ownership, education, and collaboration.
```

Comment:
So are you thinking about participatory design in [the Scandinavian sense](https://en.wikipedia.org/wiki/Participatory_design)? That's usually what I associate with the term. A [classic article](http://www.watson.ibm.com/cambridge/Technical_Reports/2010/TR2010.10%20Participatory%20Design%20The%20Third%20Space%20in%20HCI.pdf) by Michael Muller of IBM Research goes into much, much greater depth.

Reply:
I did a fair bit of background reading — though none of the ones you're suggesting; thank you for those! — but I'm not thinking of anything other than documenting how 18F put participatory design into practice with our partners. What are the essential elements?